debian-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Installs all of the necessary packages in a Docker container
+# MUST BE RUN WITH ROOT
+
+apt-get update
+apt-get upgrade -y
+apt-get install -y bc binutils binutils-aarch64-linux-gnu binutils-arm-linux-gnueabi bison ccache curl flex expect gcc git gnupg libssl-dev openssl make qemu-system-arm qemu-system-x86
```

Comment:
Do you mind putting these one one line each separated by backslashes? It would make it easier for me to read.

Do we need GCC? 😈

Reply:
I see @shenki put them on separate lines, too. 🥂

Reply:
> Do you mind putting these one one line each separated by backslashes? It would make it easier for me to read.

Yes.

> Do we need GCC? 😈

Probably not, will remove :P

Reply:
Addressed in 18515a3d90b68c06da043f1459e01e45670afe22